packages/@okta/vuepress-site/quickstart-fragments/widget/default-example/index.md

```diff
@@ -73,51 +72,50 @@ Then copy this widget configuration into your front-end application:
 ```html
 <script type="text/javascript">
   var oktaSignIn = new OktaSignIn({
-    baseUrl: "https://${yourOktaDomain}",
-    clientId: "{clientId}",
+    baseUrl: 'https://{yourOktaDomain}',
+    clientId: '{yourClientId}',
     authParams: {
-      issuer: "https://${yourOktaDomain}/oauth2/default",
+      issuer: 'default',
```

Comment:
This behavior of passing only an authorization server ID (which is expected to be combined with the `baseUrl`) is not supported in the next major version of AuthJS, which is coming soon: https://github.com/okta/okta-auth-js/pull/326

This is because we have eliminated the `url` option from AuthJS, and issuer is a required option. Probably best to stick with the full issuer in the sample.